db/schema.rb

```diff
@@ -0,0 +1,62 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2023_04_05_054637) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "comments", force: :cascade do |t|
+    t.text "text"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "author_id", null: false
+    t.bigint "post_id", null: false
+    t.index ["author_id"], name: "index_comments_on_author_id"
+    t.index ["post_id"], name: "index_comments_on_post_id"
+  end
+
+  create_table "likes", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "author_id", null: false
+    t.bigint "post_id", null: false
+    t.index ["author_id"], name: "index_likes_on_author_id"
```

Comment:
- [ ] It would be better if we make a few changes here in order to fulfill the new requirements. Let's take note that the only foreign key `author_id` must exist between `users` and `post` only. The other relationship between users and the other tables must maintain the `user_id`. Thank you :man_technologist: 
![Screenshot from 2023-04-05 08-35-23](https://user-images.githubusercontent.com/46388113/230026707-aedbeb03-e806-435b-8df4-9cc523ff3619.png)

Reply:
Thanks, bro a lot for your advice, but can you please clarify what you want to say? if possible give me some guidance please,  it looks like I didn't understand what you want to suggest to me.